modules/sagemaker/sagemaker-model-cicd/test/__snapshots__/model-build-code-pipeline-stack.test.ts.snap

```diff
@@ -659,7 +659,18 @@ exports[`ModelBuildCodePipelineStack to match snapshot 1`] = `
             {
```

Comment:
Just confirming if we need the snapshots?

Reply:
The tests check against them. I honestly didn't look deep into the tests, besides fixing this up.